vslib/src/sai_vs_fdb.cpp

```diff
@@ -156,10 +206,21 @@ sai_status_t internal_vs_flush_fdb_entries(
              * data base.
              */
 
+            fi.bridge_port_id = it->second.at("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID")->getAttr()->value.oid;
+
+            SWSS_LOG_INFO("FDBDelNotification: fdb_entry:%s bpid:%s", sai_serialize_fdb_entry(fi.fdb_entry).c_str(), sai_serialize_object_id(fi.bridge_port_id).c_str());
+
             it = fdbs.erase(it);
+
+            sendFdbDelNotification(fi);
         }
     }
 
+    /* Sending notification for each fdb entry individually above, as a single
+     * response causes syncd to delete ASIC_DB entries in a loop  and
+     * stuck for long time when a large number of FDB entries are present.
+     */
+#if 0
```

Comment:
still dont know why this is disabled

Reply:
sendFdbDelNotification() will notify sai redis lib notification thread and orchagent, so there is no need to do meta_sai_on_fdb_event().

Reply:
there is need to call that, please analyse whats happens in that function meta_sai_on_fdb_event

Reply:
Got it. This meta sai update is for the vs sai. I just disabled the flush notification part now, as notification is added for each individual fdb. Thanks.